tests/test_activation_timeout.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module contains the tests for the activation timeout class
+"""
+from spokestack.activation_timeout import ActivationTimeout
+from spokestack.context import SpeechContext
+
+
+def test_timeout():
+    max_active = 500
+    context = SpeechContext()
+    timeout = ActivationTimeout(max_active=max_active)
+    context.is_active = True
+
+    for i in range(max_active + 1):
+        timeout(context)
+
+    assert not context.is_active
```

Comment:
there are more cases that should be tested here - check the logic in the component's `__call__` function and verify that all of the conditions are being exercised